### PR TITLE
CCD-1263 CVE-2021-22118 in CCD Data Store API

### PR DESCRIPTION
**Pull request by skandala-cgi**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/CCD-1263
https://tools.hmcts.net/jira/browse/CCD-1264
https://tools.hmcts.net/jira/browse/CCD-1265

### Change description ###
CVE-2021-22118 in CCD Data Store API

**Does this PR introduce a breaking change?** (check one with "x")
```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by MSancaktutar — APPROVED** [review]
LGTM

**Review by MSancaktutar — APPROVED** [review]
LGTM

**Review by MSancaktutar — APPROVED** [review]
LGTM

File: build.gradle [hunk 1]
```diff
@@ -1,6 +1,6 @@
 buildscript {
     ext {
-        springBootVersion = '2.3.4.RELEASE'
+        springBootVersion = '2.4.4'
```
Comment (danlysiak) [inline build.gradle:3]: ⚠️ Updating to Spring Boot 2.4 will require further changes to move to the reading of secrets (same changes as per Slack discussion).
https://spring.io/blog/2020/08/14/config-file-processing-in-spring-boot-2-4

Comment (danlysiak) [inline build.gradle:3]: ⚠️ Updating to Spring Boot 2.4 will require further changes to move to the reading of secrets (same changes as per Slack discussion).
https://spring.io/blog/2020/08/14/config-file-processing-in-spring-boot-2-4

**Review by danlysiak — CHANGES_REQUESTED** [review]

**Comment by skandala-cgi** [comment]
Raised a new one

**Pull request closed without merging**